_posts/2020-10-04-newpipe-0.20.0.md

```diff
@@ -1,154 +1,176 @@
 ---
 layout: post
-title: "NewPipe 0.20.0 released: A new major version, a new unified player, a new notification experience... A lot is new this time!"
```

Comment:
That is a tripledot instead of an ellipsis, and four "a" in a row.

Reply:
Yup. Should have been an ellipsis.

Why is it a poor choice of words?

I didn't like "A lot is new this time!" but I couldn't think of a good replacement either.